### DISCUSSION TITLE
More presets I made

### DISCUSSION THREAD

**Discussion by rekkol** [ntsc-rs/ntsc-rs#231] [Presets]
More presets I made, This time for Television. I finally learned that less is more.

I originally had them labeled in terms of decades (i.e "1970s tv") but then I realized that their all random, and they could apply to any decade, Also created an extra VHS preset, and numbered them all (i.e "VHS (1)" or "Color TV (3)")  And I was very OCD about them looking realistic so I hope their good, They look good to me at the moment,

All of them are Interlaced with lower field first (Idk if that's important but yea)

Here's the preset folder (Nicely sorted out and everything) 

(Example photos coming soon)

[NTSC-RS PRESETS.zip](https://github.com/user-attachments/files/17716155/NTSC-RS.PRESETS.zip)

**Comment by sakhalif456** [comment]
ok

**Comment by dreamsynth** [comment]
They are very good! Thank you!
